### PR TITLE
feat: add download study button

### PR DESCRIPTION
**Pull request by pedrokohler**
Adding button to download study/series:

image

Solves https://github.com/ImagingDataCommons/idc-slim/issues/110

### CODE REVIEW

**Comment by pedrokohler** [comment]
@fedorov please take a look

File: src/components/DownloadStudySeriesDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                </div>
+              </>
+              )}
+        </div>
+      </section>
+    </div>
+  )
+}
+
+const getStudyAndSeriesInfo = (): {
```
Comment (igoroctaviano) [inline src/components/DownloadStudySeriesDialog.tsx:97]: Please add js docs/types for every function

File: src/components/DownloadStudySeriesDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+import { PaperClipOutlined } from '@ant-design/icons'
+
+const getConfig = (): {
+  description: string
+  instructions: Array<{
+    command: string
+    label: string
+  }>
+} => {
+  const defaultConfig = {
```
Comment (igoroctaviano) [inline src/components/DownloadStudySeriesDialog.tsx:11]: Can we expose this to slim's configuration? so then we can customize these values without having to build the viewer again

**Review by igoroctaviano — CHANGES_REQUESTED** [review]

**Comment by fedorov** [comment]
Looks good to me, thank you!

@pedrokohler I assume, similar to OHIF, it is not trivial to hyperlink idc-index in this popup, right?

**Comment by pedrokohler** [comment]
> Looks good to me, thank you!
> 
> @pedrokohler I assume, similar to OHIF, it is not trivial to hyperlink idc-index in this popup, right?

It's the same thing: doable in an hour or so, because the content of the dialog is variable, so we must parse the config strings to display the hyperlink.

**Comment by fedorov** [comment]
Let's go ahead as is then! We can address both at a later time to add hyperlink processing.

**Review by igoroctaviano — APPROVED** [review]
LGTM

**Pull request merged**